tensorflow/contrib/lite/examples/label_image/BUILD

```diff
@@ -0,0 +1,39 @@
+package(default_visibility = [
+    "//visibility:public",
+])
+
+licenses(["notice"])  # Apache 2.0
+
+load("//tensorflow:tensorflow.bzl", "tf_cc_binary")
+
+tf_cc_binary(
+    name = "label_image",
+    srcs = ["label_image.cc"],
+    deps = [
+        "//tensorflow/contrib/lite/kernels:builtin_ops",
+    ],
+    linkopts = select({
```

Comment:
This could simply be: 
  linkopts = tflite_linkopts()
and you need to:
  load("//third_party/tensorflow/contrib/lite:build_def.bzl", "tflite_copts")

Reply:
I guest you mean  `load("//tensorflow/contrib/lite:build_def.bzl", "tflite_linkopts")` instead of `load("//third_party/tensorflow/contrib/lite:build_def.bzl", "tflite_copts")`. That also works. However, `/tensorflow/contrib/lite/kernels:builtin_ops` needs `libm`, so I'll use ` linkopts = tflite_linkopts() + ["-lm"]`